partner/WHMCS/modules/servers/jetpack/jetpack.php

```diff
@@ -183,14 +187,18 @@ function make_api_request($url, $auth='', $data=[])
     ));
 
     $response = curl_exec($curl);
-    if (curl_error($curl)) {
+    $http_code = curl_getinfo($curl, CURLINFO_HTTP_CODE);
+
+    if ($http_code >= 400) {
+        logModuleCall('jetpack', __FUNCTION__, $url, $data, $response);
+        throw new Exception('Something Went wrong');
```

Comment:
@nb I've updated this as well based on the logModuleCall feedback from earlier 🙂. I had added more in a later commit to let the user know to enable Module Logging as this error message wasn't great but am now returning an error string indicating what likely went wrong when making the API request and returning this as the error string so it can be logged in the Activity Logs.